attractions/dropdown/dropdown-context-key.js

```diff
@@ -0,0 +1,2 @@
+export const CONTEXT_GET_DROPDOWN_SHELL_BOUNDARY = 'getDropdownShellBoundary';
```

Comment:
I don't think we need the `CONTEXT_` prefix. Also, it's preferable to use `{}` as a value of a context key because no two objects are referentially equal to each other, so we don't risk collisions with other Svelte code. Also I think context keys look better in simple camelCase.
```suggestion
export const getDropdownShellBoundaryKey = {};
```